README.md

```diff
@@ -272,3 +271,16 @@ curl -I 'http://localhost/api/users/1.json' -X POST -d '{ "user": { "id": 1 } }'
 | Parameters.file                | File, StringIO, Rack::Test::UploadedFile, ActionDispatch::Http::UploadedFile or subclasses |
 | Parameters.decimal(8,2)        | value is a String, Integer or Float with a precision of 9 and scale of 2                   |
 | Parameters.hex                  | value is a String that matches the hexadecimal format |
+
+## Development
+
+### Releasing a new version
+
+```
+git checkout master && git fetch origin && git reset --hard origin/master
+bundle exec rake bump:<patch|minor|major>
+git tag v<tag>
+git push --tags
```

Comment:
I would recommend using `rake release` instead. Just set `ENV['gem_push'] = 'false'` in the Rakefile. See https://github.com/zendesk/active_record_host_pool/commit/59eb7c0a23308c789311fadcc06b315733dc92a5 for inspiration.